docs/tutorials/build_a_node_in_rust.md

```diff
@@ -721,7 +723,7 @@ let persister = DataPersister { data_dir: ldk_data_dir.clone() };
 
 **Implementation notes:** if the `ChannelManager` is not persisted properly to disk, there is risk of channels force closing the next time LDK starts up. However, in this situation, no funds other than those used to pay force-closed channel fees are at risk of being lost.
 
-**Dependencies:** `ChannelManager`
+**Dependencies:** `ChannelManager`, `NetworkGraph`
 
 **References:** [`Persister` docs](https://docs.rs/lightning-background-processor/*/lightning_background_processor/trait.Persister.html)
 
```

Comment:
On line 736 below, it looks like we're missing the `net_graph_msg_handler` parameter to `BackgroundProcessor::start`. Probably has been missing for awhile. 😬

Reply:
Updated. My understanding is that it is optional though?

Reply:
Yes, optional but would need to pass `None` if not needed.